### ISSUE TITLE
Propose Mockups for Official Site

### ISSUE THREAD

**Issue by hscstudio** [yiisoft-contrib/yiiframework.com#72]
I only propose for new official site

![yii2 official site](http://www.hafidmukhlasin.com/wp-content/uploads/2015/11/mockup-466x1024.png)

http://www.hafidmukhlasin.com/2015/11/26/yii2-rebranding-yii-framework-official-site/

**Comment by jacmoe** [comment]
It looks good, but I think that - given the current stage - it would be better if you helped out with the current effort - we are bit beyond mockups and concepts :)

You could either fork or interact more indirectly.
However, good ideas, and I will probably steal something from it. Thanks :)

**Comment by samdark** [comment]
By @eshill:

![index](https://cloud.githubusercontent.com/assets/47294/11420550/e31d662e-943f-11e5-9e6f-1aba72b706a1.png)

**Comment by rob006** [comment]
@samdark This looks really cool. Download button could be bigger. Somewhere nearby should be also a button for GitHub repo. But apart from that, it looks really great and contains all the important information.

**Comment by samdark** [comment]
@jacmoe what do you think?

**Comment by jacmoe** [comment]
I think it is a bit on the busy side, but it definitely has some niceties :)
I think the best time to think about integrating would be not until I've
completed the overall skeleton.
A day or two, I think.

**Comment by samdark** [comment]
OK.

**Comment by jacmoe** [comment]
It is really, really nice :)
I don't want to dismiss it - but I am slightly preoccupied at the moment.
Does @eshill have a working prototype or is it a pure graphical mockup?

**Comment by SilverFire** [comment]
@eshill preview looks promisingly

**Comment by samdark** [comment]
@jacmoe he did working prototypes for yiiframework.ru: http://eshill.ru/yii/project_view.html so he probably can do it this tme as well.

**Comment by jacmoe** [comment]
Good to know :)

This is the front so far:
![yiilogo15](https://cloud.githubusercontent.com/assets/85424/11429551/f6f89dfa-9479-11e5-89f7-57233065332f.png)

**Comment by jacmoe** [comment]
There will be a Getting Started section under the 3 column feature section (and the columns of that section will be made same height) with 3 steps to success.
Then there will be two colums with stuff (news?)
Powered by Yii (show case)
And testimonials.

After that, ,I think it's time to do something about the Guide/API sections..

**Comment by muhammadcahya** [comment]
I'm so sorry if disturbing euphoria of bootstrap
What do you think about zurb foundation (for the site) ?

For now [Bootstrap 4](http://v4-alpha.getbootstrap.com) still an alpha, but new [zurb foundation 6](http://foundation.zurb.com) has started the final version now, and can be used or implemented right now.
Yes both Bootstrap or Foundation is good, but foundation is not mainstream (I think), more lightweight, and have a good design pattern concept.
How about this, what's you're oppinion ?

---

I apologize for example the design of the above, according to the website I think must show the identity, especially of the logo (representing the colors of the logo, or one dominant color), so that the logo and the website can be aligned and harmonized.

But I'm so sorry, with respect to who makes, design the above in my opinion was bland and boring in terms of color, do you also think so ?

I do not have much time to make a sketch from zero, if there who can give me psd, or any other that I can edit ?

**Comment by jacmoe** [comment]
I personally prefer Foundation - and I am excited about Foundation 6 !
It is really powerful.

But, Bootstrap 4 is actually usable right now, and I almost can't resist the temptation to use it!
It is now based on Sass, and they have cleaned up the css a whole lot and made it much easier to override all the things :)
I especially like the new Bootstrap cards..

**Comment by jacmoe** [comment]
Do you really think that my screenshot was boring? :cry:

**Comment by muhammadcahya** [comment]
I'm so sorry, yes I think so :(
But that's just my opinion, probably maybe others people more like it :)

**Comment by jacmoe** [comment]
If you are a little bit bored, I guess I am happy with that.
Because my goal is to avoid making the site too busy - I want the content to be in focus.
It should not be too busy. :)
I am not sure if I am able to achieve that goal, but I am trying.
Right now I am making sure that it is easy to change all settings: fonts, colours, layout.. so that we eventually can get a site that we are happy with.

**Comment by muhammadcahya** [comment]
I'm sorry I did not post mockup yii sites, but hopefully this design can refresh your ideas and inspiring.
Sorry if there would be long scroll :p

![Design 1](https://cloud.githubusercontent.com/assets/4241620/11431904/a420faf4-94d4-11e5-9815-e5011297b6ff.jpg)
![Design 2](https://cloud.githubusercontent.com/assets/4241620/11431902/a404287a-94d4-11e5-89d4-ee8e0827e670.jpg)
![Design 3](https://cloud.githubusercontent.com/assets/4241620/11431907/a4352182-94d4-11e5-9391-8077588c8446.png)
![Design 4](https://cloud.githubusercontent.com/assets/4241620/11431905/a4236d16-94d4-11e5-9b0e-3f593a605b17.jpg)
![Design 5](https://cloud.githubusercontent.com/assets/4241620/11431903/a4121be2-94d4-11e5-91f9-13033b8446c4.jpg)
![Design 6](https://cloud.githubusercontent.com/assets/4241620/11431908/a45996ca-94d4-11e5-8aa1-0c3381ec3523.jpg)
![Design 7](https://cloud.githubusercontent.com/assets/4241620/11431909/a46a038e-94d4-11e5-957f-7dd6ab0aec7b.png)
![Design 8](https://cloud.githubusercontent.com/assets/4241620/11431910/a470fc48-94d4-11e5-9954-dba357eab921.png)
![Design 9](https://cloud.githubusercontent.com/assets/4241620/11431911/a477921a-94d4-11e5-8978-e427bf70c2db.png)

@ regards to author / designer this mockup

**Comment by jacmoe** [comment]
Wow - that's a lot of concepts :)

To be honest, it looks like that many of todays designs are made by girls..

I prefer a more 'manly' design.

There are interesting design patterns, though.

**Comment by muhammadcahya** [comment]
Girls or 'manly' design I think it's about color @jacmoe
I only refresh about 'concepts', layout, and positioning with design above, because I'm busy at the moment, so for now I can't give me an idea about the sketch mockup
You can chose 'manly' colors for 'manly' design @jacmoe, it's up to you

Sorry, Do you think your mockup has are already 'manly' design, the colors you chose already a 'mainly' colors ?

But i noticed here, Yii for All People, All Person, *not separate for one/specific gender, whether yii only for Men* ?

**Comment by hscstudio** [comment]
I think some designs that proposed by @muhammadcahya have same pattern in color.. its use soft color.. its great and modern.. it's not about girly or manly.. please use soft blue.. current design https://github.com/yiisoft-contrib/yiiframework.com still use dark blue.. :)

**Comment by jacmoe** [comment]
Dude you are too serious!
You must admit that the preference for pastel colours that a lot of people
have these days are not exactly what you'd normally associate with
manliness.
Take a look at Phalcon homepage - really awful design!
Don't attack me or my views when you don't know either -I have been civil
towards you.

Anyway,since we already are working on the new site,I think we need to be
more specific.
Otherwise this topic is wasting time...

**Comment by muhammadcahya** [comment]
@jacmoe I'm not serious, maybe you are responding to this too seriously :)
With all my respect to you, I do not mean to attack you

I always appreciate your opinion as well as any person, my comment above just my opinion, just my advice about mockup, I'm so sorry if you think my words is wrong or as attack

Best Regards

**Comment by jacmoe** [comment]
It would be a big mistake if the new Yii homepage ends up looking like a
thousand other websites.
That is not the goal.
The goal is to be bold and to grab peoples attention.

Soft and washed out colours are great for social hubs, newssites, and the
like, IMO.

**Comment by jacmoe** [comment]
Anyway, when I am done doing the initial theme, then you are free to change
the blue to a light cyan if you like :)
Hopefully I have made it very easy to change the looks.

**Comment by jacmoe** [comment]
Muhammad: peace :)
I think that we might be direct people both of us - do not worry about it :p

**Comment by jacmoe** [comment]
I can make a version of the front tomorrow without the blue, but I can't
make a softer or lighter blue. That would introduce a fourth color..
The palette is Yii green, Yii blue and Yii yellow and then various whites
and greys.

**Comment by hscstudio** [comment]
@jacmoe:
I appreciate Your work and design, but I very love Yii too, 
so permit me to correct Your mockup..

![yii mocakup](https://cloud.githubusercontent.com/assets/2976897/11467303/b2384758-9779-11e5-8852-bbb700ad8bb5.png)

**Comment by jacmoe** [comment]
Gray does not work for the featured section, so I used blue..

You can see the updated front page here:
![local yiiframework com-1280x768](https://cloud.githubusercontent.com/assets/85424/11470302/d56c732e-975c-11e5-88a0-790d19439614.png)

**Comment by jacmoe** [comment]
Since pageres did not handle the shadow filter:
![highlight3](https://cloud.githubusercontent.com/assets/85424/11470360/30dc91ee-975d-11e5-89dd-b6ade4344a30.png)

**Comment by jacmoe** [comment]
I tried gray for the powered by Yii section, but stay.com is white so it disappeared when I applied a grayscale filter on it.
I kind of like the blue..
The colours can always be changed. That is not my current focus right now - just making sure that the content gets in and looks good together.

**Comment by hscstudio** [comment]
You have do significant improvement.. I like it (y) 
But I have some added..
1. Please see @eshill design... maybe you can combine it.. they are: color of panel menu.. I prefer blue dark instead of black; add more content in the middle.. for example: book, video etc..
2. For footer.. I prefer You use my first mockup.. start form TESTIMONIALS until bottom..
3. You dont force all image as svg.. it's difficult you may use image..

**Comment by samdark** [comment]
@jacmoe yes, I'll update Stay.com logo. I have a vector source for it.

**Comment by jacmoe** [comment]
@samdark I already updated it from the svg :)

**Comment by jacmoe** [comment]
Thanks @hscstudio :)
Yes, there will be more sections on the front page.

Good idea about the navbar - I'll try a dark blue. And look more at the @eshill design, when I am ready.

I won't use svg's when I am done because regular png/jpgs are smaller and more widely supported - it's just easier to test things out when you have images that can take any size easily :)

**Comment by jacmoe** [comment]
This is a test with a blue navbar:
![highlight4](https://cloud.githubusercontent.com/assets/85424/11478615/53967836-978e-11e5-9ced-e750829cd034.png)

**Comment by samdark** [comment]
Looks better IMO.

**Comment by jacmoe** [comment]
It grows on you, yes. :)

Will probably change the top-bar to the same grey colour as the footer - let's see..

We still have to decide on what font/logo to use - I am happy with Josefine and Roboto, but ..

**Comment by jacmoe** [comment]
Dark topbar:
![yiisite3](https://cloud.githubusercontent.com/assets/85424/11480659/edff24e0-9798-11e5-8e0c-a594dca3c5fb.png)

**Comment by samdark** [comment]
Much worse.

**Comment by jacmoe** [comment]
Yes - maybe the previous top-bar colour was better?

**Comment by samdark** [comment]
Absolutely.

**Comment by samdark** [comment]
Current (live website) logo is OK overall. We can try removing gradients from it so it looks flat and "modern" but to me it's good as is. As for "yiiframework" text, I'm not sure what to do with it. To me, again, it's OK. See @eshill mock on how it may look like.

The font overall could be changed but the decision is to be made based on how guide/API text looks with that text.

**Comment by jacmoe** [comment]
I like our current logo - the petals part of it :)

Right now we are using Josefine for the headers - and logo - and Roboto for everything else (I think).

It seems to look good with the guide as well.

Josefine is pretty close to the font in the current logo. I will check out @eshill s mock.

**Comment by jacmoe** [comment]
I changed the color for the topbar links to a dark grey:
![yiisite4](https://cloud.githubusercontent.com/assets/85424/11481007/1279dfa2-979b-11e5-9156-b788e99ae973.png)

I think that looks better than the blue (default) links.

**Comment by SilverFire** [comment]
I think the last is the best in the thread. However, we still can make it better. @jacmoe I'm highly appreciating your work, thank you very much! I'm not a UX/UI man, so don't treat my feedbacks as pro-advises.

I think black word "framework" might be lightened.

**Comment by SilverFire** [comment]
Update: Okay, it's not black, it's gray, but my shity TN-display on laptop makes it black :(

**Comment by jacmoe** [comment]
I have thought the same thing @SilverFire - yes: the word 'framework' ought to be lighter.

It is too 'gloomy' right now.

And, perhaps - when it's lighter - I will see if changing the blue 'framework' in nav-bar logo to the same grey will work.

I am no UX guru either - it has always been a weak spot of mine. That's why I grabbed this opportunity to level up ;)

**Comment by jacmoe** [comment]
I changed the grey:
![yiisite5](https://cloud.githubusercontent.com/assets/85424/11485692/f95b667a-97b4-11e5-827a-7052aba27c64.png)

Also, changed the colour of 'framework' in nav-bar - I have to think about this, I think.. ;)

**Comment by hscstudio** [comment]
![yii mocakup 2](https://cloud.githubusercontent.com/assets/2976897/11488014/df28fefc-97f5-11e5-9d4d-f6fd34b1c3b3.png)
@jacmoe : You add many great improvement. and  I add some added
1. You should not repeat yii logo on main section, Yii logo enough in navbar
2. The use of font on the main part doesn't have to be similar to font of logo. Use white color.. and font case capitalize in each word. then add shadow.. because its be focus of visitor
3. Give more space after and before this main part.. keep visitor to focus.. and give memories impression... dont make visitor read much..
4. Sub Title "the solid foundation for your php application" I prefer use lowercase and should not end with dot (.). see our picture..
5. Add button in the third line.. after visitor read Yii Framework and know that "the solid foundation for your php application" .. what that he do? Yes.. he click get started..

Keep "shock" in first impression.. I think You know.. what I mean

Thank You.. I love Yii

**Comment by jacmoe** [comment]
Right now, all the information the user needs to know is above the fold.

I am not a big fan of the teaser approach.

And 'PHP' should never be written in lower-case - that's just wrong! :P

I don't personally like the spotlight effect - it get's old real soon.

But, apart from that, a good idea.

Not sure if I am willing to go that route, though.

Let other people chip in. :)

**Comment by jacmoe** [comment]
You do have a point about the duplicate Yii logo, though - or I think that you might have.

I would rather hide the logo in the nav-bar on the front page, then.

How does that sound?

((I will try and test it out ...))

**Comment by hscstudio** [comment]
I use lowercase on text "the solid foundation for your php application" only for art.. I know should PHP in uppercase... but art break all.. depend our imagine.. But if we disagree.. I think we can change PHP to web.. think globally.. that Yii is web application not only php application... "the solid foundation for your web application"

I am not a teaser approach fan.. but we can do for visitor focus.. and give simple impression for visitor.. You can see my picture not realy have spot corner.. I know about, spot corner can make old if it only small and have same diameter.. but 
my picture.. no.. soft color spray.. and focus in center..

About "hide the logo in the nav-bar".. sound good.. but we need try it.. I will always support this project..

You are not angry? I am sorry if I disturb You work..

I am only Indonesia Yii Lovers :).. and I want Yii get populer, trendy and modern

**Comment by jacmoe** [comment]
I am not angry - I know that it is quite common to have a minimal and teaser like header - what you have shown is a good idea, but very similar to a thousand sites - no disrespect, just my opinion, OK?

I wanted to go along a different route, where pretty much everything that can be said about Yii is visible without scrolling or clicking.

The bold colour is also a choice that I made..

And regarding the spotlight effect: actually, it does have a spotlight effect right now, because the white 'yii' next to the logo really draws your attention - it does create a spotlight effect, but I think it is more subtle and perhaps more effective?

I don't know :)

Thanks for the input - it is appreciated - I just need to express my intentions, so don't take it the wrong way.

I am Danish - we are known to be direct in a way that could be confused with rudeness. :p
((Vikings...))

I am open to any change - as soon as I have done what needs to be done - you are free to change anything.

**Comment by hscstudio** [comment]
![yii mocakup 2](https://cloud.githubusercontent.com/assets/2976897/11489826/5b9b3784-9805-11e5-9f21-b23e7029c433.png)
or
![yii mocakup 2](https://cloud.githubusercontent.com/assets/2976897/11489852/8daa2eec-9805-11e5-851d-c067ae21026f.png)

oke.. You (y).. I propose my main part of our mockup.. and we can hide logo in navbar..

**Comment by muhammadcahya** [comment]
With all my respect,
I like all the mockup/design above , this is my idea, what's your opinion all ?

My Half(50%) Mockup
![mockup](https://cloud.githubusercontent.com/assets/4241620/11489885/e9f11256-9805-11e5-8e44-8121d7ca5fc6.jpg)

**Comment by jacmoe** [comment]
@hscstudio 

That's how it looks in mobile mode:
![yiilogo24](https://cloud.githubusercontent.com/assets/85424/11489891/c72176c6-97d3-11e5-8314-494ddf98fad3.png)

Let us think about what you suggested - we need more input.

@muhammadcahya Interesting. :) I like the top part - after that it kind of becomes bland.

It looks stylish - a pity about the logo, though.

I think I need to let this hang for a bit because I need to concentrate on my todo-list.

Don't let the comments/ideas stop.

**Comment by hscstudio** [comment]
@muhammadcahya : I think too soft.. need more character..
@jacmoe : 
in mobile 
we should hide text yiiframework.. leave Y logo only in navbar

Please dont use text "framework" with grey color.. its dead color with blue... Should white color for it same with text Yii

**Comment by muhammadcahya** [comment]
@jacmoe & @hscstudio : Yes, I think bland and too soft so
Like I said before, this is not finished yet, maybe 50%, I'm not playing about the color now, but in design above I talk about 'layout' concept :)

**Comment by jacmoe** [comment]
@hscstudio :

Nope. 'Yii' is more important than 'framework' :)

Well, I might change it later - not now, though. I don't agree that grey and blue are dead together.

Not sure about the hiding of the logo - it would look weird. At least that's what I think.

I have to follow my idea through - otherwise, if I should change direction completely all the time, I wouldn't be able to create a coherent design, which I think is important: that all the pieces fits together to form a whole.

What happens later, who knows ? :)

**Comment by hscstudio** [comment]
I hope that you immediately change mind @jacmoe :+1: 

@muhammadcahya : I think layout more simple that color :) Its can change regularly.. but color NO.. :) good try

**Comment by jacmoe** [comment]
@hscstudio :+1:  :dancer: 

I might drop the 'yiiframework' for the jumbotron section, just for kicks.. no promises, though :)

Fortunately, the way I am coding the colours can be changed just by changing a few sass variables.

**Comment by jacmoe** [comment]
About being minimal or not:

Bootstrap can get away with a purple background with a 'B' simply because it is universally known.

Yii is popular, but there are still a lot of (unfortunate) people that don't know what 'Yii' stands for. So they need a bit of explanation/persuasion.

Bigger brands don't need to do that. They are brands.

**Comment by samdark** [comment]
@hscstudio good suggestions, you definitely know what you're doing. Keep these going. Thanks!

@jacmoe hiding logo in navbar at start page would make it inconsistent because at pages that aren't front page you have to show the logo in the bar.

I also think that grey and blue aren't looking goog enough together.

I love https://github.com/yiisoft-contrib/yiiframework.com/issues/72#issuecomment-160817814 second variant or https://github.com/yiisoft-contrib/yiiframework.com/issues/72#issuecomment-159868139 step by step features description w/ screenshots.

@muhammadcahya good try but overall I would not go this direction.

**Comment by jacmoe** [comment]
I separated the words and gave 'framework' just a tiny tint of grey:
![yiilogo28](https://cloud.githubusercontent.com/assets/85424/11499843/5ffc8d72-9829-11e5-993d-60f84c0824c9.png)
I also made the background slightly darker - still ends up being slightly lighter than base Yii blue at the end.

**Comment by samdark** [comment]
Looks much better.

**Comment by muhammadcahya** [comment]
I not want to show on my mockup again :p
Yes, I gracefully as @samdark say, overall will not follow my direction :)

![mockup](https://cloud.githubusercontent.com/assets/4241620/11504623/da4e48f0-9878-11e5-9a2a-232b559c8f44.jpg)

But here, with this mockup above, I want to show. on the word 'yii framework', is bad with 'Y' big ? Why 'Yii' with a capital 'Y' looks weird ?
Like the logo menu in my example above, i think with 'Y' is same better with 'y', but with 'YiiFramework' I think more readable than 'yiiframework', what's your opinion ?

With a balance between the logo size, spacing and text, i think will be better :)

**Comment by jacmoe** [comment]
It just looks weird next to the logo, that's all - not on it's own :)

I like your mockup @muhammadcahya , but not for this project. Perhaps for other things?

**Comment by hscstudio** [comment]
Poin plus of @muhammadcahya last mockups are: 
1. he create big navbar (height more than generaly) and white color.. yii logos very match with soft color background.. why? Because yii logo have 3 colors.. I think it is modern design for navbar.. 
2. He is not repeat logo in main section and navbar.. because repeat is can make bored.. but if we want show logo yii in main section.. we can hide its logo in navbar only on homepage and show when other page... many website do it..
3. Blue section with 2 button are download and take tour.. its good approach..  he guide user to do action.. "okey.. yii good but what next? What can I do? Please take action now"
4. Teks YiiFramework its unity and our brand... we cant break yii and framework..  teks "Yii Framework" good to... instead of lowercase

But for now.. @jacmoe have do many time and progress we should give help him to improve current project not create one new from scracth :)

I know that current design is anti mainstream.. and @muhammadcahya design is mainstream but we should remember that dark color.. and many text in first impression is mainstream in old trend design.. 

Now.. trend design is big, simplicity & softcolor

But forget it.. if we want force to anti mainstream..

My advice.. follow your feel.. You feel "I like your mockup @muhammadcahya" not mainstream or anti mainstream..  

If we use dark color.. we should show a fantastic anything.. maybe with briliant logo.. but for safe we should use soft color.. okey yii have blue brand.. but soft blue.. 

Use universal looks forexample.. @eshill design good.. for me, for you, for samdark, and other.. my indonesian community like it too.. 

Follow Your feel @jacmoe .. I know You should pro..

Hem.. forget myadvice.. do best job... 

We give to you opportunity to make official yii looks good..

#I am only another yii lover

**Comment by eshill** [comment]
@hscstudio thanks!

**Comment by jacmoe** [comment]
Yeah, great work @eshill :)

However, I need to follow my train first before thinking about integrating ideas.

As for style, I am a huge fan of Mozilla.org's new site - it really moves me with it's strong, vibrant colours and sophisticated elements - I don't forget that I visited it:
![mozillasite](https://cloud.githubusercontent.com/assets/85424/11525133/d4903542-98d0-11e5-9a98-c70b34167371.png)

**Comment by jacmoe** [comment]
I really appreciate your thoughts @hscstudio - thank you!

I really, really hope that I can do Yii and the awesome community justice :)

**Comment by samdark** [comment]
@jacmoe these three are OK but not as strong as @eshill idea.

**Comment by jacmoe** [comment]
There are six - 'Relational and noSQL databases', 'Great tools' and 'Exceptional community' are buttons too.

'Download Yii' would just lead the user on a goose chase - I think it's better showing the command that you need to run to get a Yii project :)

I can put a big download button there but it would just lead to the Quick Start (tour) page or something.

Perhaps later.

**Comment by albaraam** [comment]
Hi all, I have some thoughts I  want to share with you guys.

First of all I wonder why did you guys start with front-end development?
I suggest (following content first approach) to first have couple of designs for the new website, then we can make a simple poll (lets say for one week) to let the community participate in choosing which design is the best (more opinions => better decision). It's even easier to make changes during this phase. 

you may be asking where can we get the designs? simply any one who knows a designer can ask him/her for a redesign of the home page (at least) in his/her spare time, and I think there will be many designers who would like to participate. And if you're okay with the idea, I can contact some designers I knew if they have time.

Actually what I want to say is that development is the last part of the process, and for design lets give it to who's expert in it.

Content ==> Design ==> Development

**Comment by muhammadcahya** [comment]
Hi @albaraam 
Thank's for sharing, very interesting :)

I think about 'following `content` is first approach', Yes already exist (in current website, etc), because it's about 'rebranding'

Actually I was curious about the designers/expert who would you contact, I want to see it :)
But what I see here, the design has been / is being made by @jacmoe , so there will probably be a bit of a clash of design
Perhaps now is to respect the @jacmoe work, and then we can give a feedback about it, all the ideas we can accommodate here

That's my opinion, waiting for the master commented :D

**Comment by jacmoe** [comment]
As far as I can see, the process has been going on for quite some time now in private - keep in mind that the repository has only been made public a short while ago. :)

They have had contact with several designers and UX experts, and have gotten some advice/suggestions, but not nearly enough.

I was dragged into it due to my previous mockup, and am now working on what is to be replacing the current site.

We are open to suggestions and contributions, but I think it is not the time for initial design brainstorming - we are working hard to get this site ready to launch.

Specific advice is welcome, not complete overhauls - otherwise we won't be able to ship in time :)

I am painfully aware that I am not a master designer, so feel free to suggest and maybe fork and do PRs.

**Comment by samdark** [comment]
The main idea now is to get content / structure right. Then if design isn't OK, make it better. If it's OK then launch and improve as it goes.

**Comment by rob006** [comment]
> Specific advice is welcome, not complete overhauls - otherwise we won't be able to ship in time :)

Is there any deadline?

**Comment by samdark** [comment]
Not really but we don't want it to take years to complete...

**Comment by jacmoe** [comment]
>Is there any deadline?

No, not that I am aware of..
But there have been plans to launch a new site for ages now, and we are all getting impatient - and the current site is not doing Yii any favors (to put it mildly) - so I think that the sooner it gets launched, the better. :)

Edit: exactly what @samdark said.

**Comment by rob006** [comment]
Yii 2.0 was released a year ago. For almost all this time there was no progress in creating the new page. Spending a week or two for real design is not such a big effort in this context. In fact, it can speed up work - spontaneous coding without any plan usually requires a lot of fixes and changes, and it is not as fast as it seems at the beginning.

**Comment by jacmoe** [comment]
What makes you think that we haven't ?

**Comment by rob006** [comment]
Because I'm not seeing any plan or final mockups? If there are any, where I can find them?

**Comment by samdark** [comment]
The plan is in issues: https://github.com/yiisoft-contrib/yiiframework.com/issues

**Comment by jacmoe** [comment]
If you care to take a look at the issues here, you will notice that the design has been discussed at length and at least on UX guy has been brought in, they have used moodboard, PDF etc., and - because the Yii team are very busy people, and UX guys tends to disappear - it all seemed to grind to a halt.

I started a topic - http://www.yiiframework.com/forum/index.php/topic/68490-new-face/ - and pretty soon I made a mockup where I presented my ideas of how a possible new Yii site would/could look like.
And later, when the source code was opened, I began to contribute.

What I saw was a rough theme and I began to integrate my ideas into it, and I think we have made good progress.

Final mockup?
I don't work that way - fortunately, I am not employed in a corporate environment where 'final mockups' are the norm.
Agile is the way to go: do work, test, get feedback, improve, test, get feedback ..

**Comment by jacmoe** [comment]
I have tried to - and continues to - make sure that it is as easy as possible to change the looks of the site by making use of good practices and good tools - think of it as a framework for this site - building blocks.
Gulp and Sass and Yii is a great combination :)
So if we don't like that blue color? No problem: change one or two variables, and the color is different all across the site.

Responsive design and prototypes is the way to go - not mockups.

https://webflow.com/blog/always-choose-prototypes-over-mockups

((Edit: I can't for the life of me remember that article that started the whole move from mockups to prototypes - I'll post it, when I find it))

**Comment by rob006** [comment]
Seriously guys, this exactly we-have-no-real-plan-just-coding replies. :) 

Currently new site code could stay closed - all work is done by collaborators, because others do not have anything to do here. As a coder I do not know how I could help, because there is no specific issues, which I could handle. If I was designer I will not spend my time for designing mockups, which probably will be ignored. You're basically close for community help.

**Comment by jacmoe** [comment]
I am not really sure what it is you are after..
All I know is that I was asked to try and take my ideas from the prototype that I made and integrate them with the site that sits in the repository we are discussing :)
I tried to start a discussion in the Yii forum, but not many people participated.

Of course, as soon as the integration got started, people have started to appear with one mockup after the other - and that is good.
Does that mean that I should stop integrating?

If you are the coding type, then a quick look at the issues would probably give some clues to what needs to be done.
If not, then do open a ticket and ask what needs to be done :)

**Comment by cebe** [comment]
> As a coder I do not know how I could help, because there is no specific issues, which I could handle.

The issues are mostly unclear because we have not defined everything in detail. E.g. user management needs to be created from scratch and we need a plan for that first.
There is a milestone for issues that need to be done for the launch: [milestones/launch](https://github.com/yiisoft-contrib/yiiframework.com/milestones/launch).
Some of them are vague because we need to find a plan for them but there are also concrete ones like #18 #46, #77. For the vague ones, you may propose solutions or discuss requirements with us.

**Comment by albaraam** [comment]
Thanks guys @jacmoe @muhammadcahya for reply, I think I got the idea as @samdark said it's all now about  getting content / structure right. Even I think (as @rob006 mentions) that it's much easier to have a design (PSD) proven by the community, then starting in development since applying a design into html/css/js doesn't take that much of time if every thing is planed and agreed.

At the end we're all Yii Lovers and have the same goal (Make Yii's website the best), keep up the good work guys.

**Comment by rob006** [comment]
>  Does that mean that I should stop integrating?

Integrating what? There is no public project of main page, so I have no idea what and how you will integrate.
I think so @eshill mockup looks much better and it's more practical than current implementation. I don't like current "Fast, Secure..." empty slogans in hero sections. @eshill slider with development steps is really great idea, far more better than current Quick Start section. Github ribbon is ugly. And putting The Definitive Guide and Class Reference into smaller navbar is ridiculous - documentation is mostly first thing what user search on framework page, and now we don't event have such a word on page. It's not just a colors, it's about the content layout, which is more difficult to change in the future.

**Comment by jacmoe** [comment]
At least you are being honest ;)

**Comment by jacmoe** [comment]
@samdark is the @eshill available in source form?
I think I am going to go for that instead.

**Comment by samdark** [comment]
Yes. I'll ask him.

**Comment by samdark** [comment]
Here it is: https://github.com/eshill/yiiframework.com

@jacmoe are you OK w/ sources in Sketch? If not, he can re-do in PSD. Also, which pages do we need additional design for?

**Comment by jacmoe** [comment]
I don't have a Mac or money :)
Does the Sketch tool export to usable formats?
Otherwise, I will just work using Colorzilla and the png.

**Comment by samdark** [comment]
OK. Will get PSD then.

**Comment by jacmoe** [comment]
I guess that none of you guys are willing to take over the design?

**Comment by samdark** [comment]
Design or HTML/CSS? @eshill would take care about design.

**Comment by jacmoe** [comment]
That sounds good. Yes, design.

**Comment by samdark** [comment]
@jacmoe https://github.com/eshill/yiiframework.com/blob/master/psd/index.psd

**Comment by jacmoe** [comment]
Nice :)

2015-12-14 10:58 GMT+01:00 Alexander Makarov <notifications@github.com>:

**Comment by samdark** [comment]
We're lucky since he's OK to work more on it polishing details and re-drawing things we don't like so if you see something outstanding, post about it.

**Comment by jacmoe** [comment]
This probably means that we say good-bye to the current logo.
Let's see.
That has been my biggest challenge so far - those 3 colours :+1:

**Comment by samdark** [comment]
Well, that was a good try.

**Comment by jacmoe** [comment]
Not at all wasted - the more you learn, the better you be.

What is his font choices?
I need to know that to use for headers, logo and body text.

**Comment by eshill** [comment]
https://www.google.com/fonts/specimen/PT+Sans

**Comment by jacmoe** [comment]
:+1: 
It has been ages since I had to work from a PSD :)

**Comment by jacmoe** [comment]
Closing because it has been decided ;)
